manual/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: manual
+title: Manual
+permalink: /:path/:basename
+---
+
+## GitHub CLI
```

Comment:
@billygriffin I added this index page called `Overview` that will load at `/manual`! Would love your thoughts and input on the content for this. I grabbed what's here right now from the repo README